indy_node/server/domain_req_handler.py

```diff
@@ -305,6 +304,18 @@ def _validate_revoc_reg_entry(self, req: Request):
             revoc_reg_def_id=req.operation[REVOC_REG_DEF_ID],
             req_id=req.reqId
         )
+
+        is_owner = False
+
+        if get_req_id(req) == req.operation[REVOC_REG_DEF_ID]:
+            is_owner = True
+
+        self.write_req_validator.validate(req,
+                                          [AuthActionAdd(txn_type=REVOC_REG_ENTRY,
```

Comment:
Here we always consider it as an ADD action, but it may be EDIT or ADD depending on if `current_entry is None`